app/assets/javascripts/admin/addon/components/value-list.js

```diff
@@ -1,17 +1,21 @@
-import discourseComputed, { on } from "discourse-common/utils/decorators";
+import { action } from "@ember/object";
+import { classNameBindings } from "@ember-decorators/component";
 import { empty, reads } from "@ember/object/computed";
+import discourseComputed, { on } from "discourse-common/utils/decorators";
 import Component from "@ember/component";
 import { makeArray } from "discourse-common/lib/helpers";
 
-export default Component.extend({
-  classNameBindings: [":value-list"],
-  inputInvalid: empty("newValue"),
-  inputDelimiter: null,
-  inputType: null,
-  newValue: "",
-  collection: null,
-  values: null,
-  noneKey: reads("addKey"),
+@classNameBindings(":value-list")
```

Comment:
Doesn't need to be addressed here, but I think this could be simplified? (that's what `:something` syntax does, right? it's been a hot minute since the last time I had to use that 😅)

```suggestion
@classNames("value-list")
```

(`:syntax` is also used in 3 other files)